### DISCUSSION TITLE
option USE_MASS

### DISCUSSION THREAD

**Discussion by AlexeiKharchev** [OpenMathLib/OpenBLAS#4037] [General]
there is option USE_MASS=1 in description (README.md)
but USE_MASS is used in Makefile.power only
I dont see nor EXTRALIBS nor COMMON_OPT in other files too

is this option unusable?
if it is true, I suggest to remove USE_MASS option from README.md

**Comment by martin-frbg** [comment]
This option _is_ specific to the POWER architecture, and to my knowledge, the IBM MASS library "only" replaces some standard math functions from libm with routines specifically optimized for their architecture. Thus there is no need for handling USE_MASS anywhere else than in Makefile.power where it enables compilation with `-ffast-math -mveclibabi=mass`  and the corresponding linker options across all of OpenBLAS.
As the option is clearly mentioned to be specific for POWER, I do not see a need for removing it from the README.
